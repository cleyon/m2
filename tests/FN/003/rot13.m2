@define aRealSymbol *With 6, you get value!*
@rot13 aRealSymbol@
@rot13 NotaRealSymbol@
