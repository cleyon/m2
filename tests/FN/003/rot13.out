*Jvgu 6, lbh trg inyhr!*
AbgnErnyFlzoby
